Cargo.toml

```diff
@@ -12,3 +12,6 @@ nom = "6.1.0"
 num-complex = "0.3.0"
 petgraph = "0.5.1"
 serde = { version = "1.0.125", features = ["derive"] }
+
+[dev-dependencies]
```

Comment:
Might users of the library want access to `insta`-generated things, too?

Reply:
Nope - these are just snapshots of test outputs